code/modules/vehicles/speedbike.dm

```diff
@@ -69,11 +520,40 @@
 
 /obj/vehicle/space/speedbike/speedwagon/buckle_mob(mob/living/M, force = 0, check_loc = 1)
  	. = ..()
-		riding_datum = new/datum/riding/space/speedwagon
+ 	riding_datum = new/datum/riding/space/speedwagon
 
 /obj/vehicle/space/speedbike/speedwagon/Moved()
 	. = ..()
 	if(src.has_buckled_mobs())
 		for(var/atom/A in range(2, src))
 			if(!(A in src.buckled_mobs))
 				Bump(A)
+
+
+//Memewagon, the less lethal more hilarious reward
+
+/obj/vehicle/space/speedbike/memewagon
+	name = "Engineering's Pinnacle"
+	desc = "The supreme department, manifest"
+	icon = 'icons/obj/bike.dmi'
+	icon_state = "speedwagon"
+	layer = LYING_MOB_LAYER
+	overlay_state = "speedwagon_cover"
+
+/obj/vehicle/space/speedbike/memewagon/Bump(mob/living/A)
+	. = ..()
+	if(A.density && has_buckled_mobs() && (istype(A, /mob/living/carbon/human) && has_buckled_mobs()))
+		var/atom/throw_target = get_edge_target_turf(A, pick(cardinal))
+		A.throw_at(throw_target, 10, 8)
+		A.Weaken(2)
+		A.adjustStaminaLoss(10)
+		A.apply_damage(rand(1,5), BRUTE)
+		visible_message("<span class='danger'>[src] crashes into [A]!</span>")
+		playsound(src, 'sound/effects/bang.ogg', 75, 1)
+		sleep(10)
+		playsound(src, 'sound/items/carhorn.ogg', 100, 1)
```

Comment:
Speedwagon already got moved out of speedbike, for the time being im going to keep them a child of speedbikes just because speedwagon is now a monstrosity whereas these are still "oversized bikes".